lib/hsa/hsaRfiTimeSum.cpp

```diff
@@ -73,21 +73,21 @@ hsa_signal_t hsaRfiTimeSum::execute(int gpu_frame_id, hsa_signal_t precede_signa
     struct __attribute__((aligned(16))) args_t {
         void* input;
         void* output;
-        // void* output_var;
+        void* output_var;
         uint32_t sk_step;
         uint32_t num_elements;
-        // uint32_t element_index;
+        uint32_t element_index;
     } args;
     // Initialize arguments
     memset(&args, 0, sizeof(args));
     // Set argumnets to correct values
     args.input = device.get_gpu_memory_array("input", gpu_frame_id, input_frame_len);
-    args.output = device.get_gpu_memory("timesum", output_frame_len);
-    // args.output_var =
-    //    device.get_gpu_memory_array("rfi_output_var", gpu_frame_id, output_var_frame_len);
+    args.output = device.get_gpu_memory("time_sum", output_frame_len);
+     args.output_var =
+        device.get_gpu_memory_array("rfi_output_var", gpu_frame_id, output_var_frame_len);
```

Comment:
since this is now an intermediate product, it can just be a `get_gpu_memory()` call.

Reply:
And it should be renamed something like `rfi_time_sum_var`, since it hasn't been input summed yet.